src/pages/CreateCirclePage/CreateCirclePage.tsx

```diff
@@ -0,0 +1,232 @@
+    <div className={classes.root}>
+      <h2 className={classes.title}>Create a Circle</h2>
+      <CreateCircleForm.FormController
+        source={undefined}
+        submit={async ({
+          captcha_token,
+          research_what,
+          research_who,
+          research_how_much,
+          research_org_link,
+          research_contact,
+          ...params
+        }) => {
+          const newCircle = await createCircle(
+            { ...params },
+            captcha_token,
+            JSON.stringify({
+              address: myAddress,
+              research_what,
+              research_who,
+              research_how_much,
+              research_org_link,
+              research_contact,
+              ...params,
+            })
+          );
+          setSelectedCircleId(newCircle.id);
+          history.push({
+            pathname: paths.getAdminPath(),
+            search: paths.NEW_CIRCLE_CREATED_PARAMS,
+          });
+        }}
+      >
+        {({ fields, changedOutput, handleSubmit }) => (
+          <div className={classes.bodyInner}>
+            <div className={classes.subtitle}>
+              Coordinape circles allow you to collectively reward circle members
+              through equitable and transparent payments. To start a circle, we
+              need just a bit of information.
+            </div>
+            <div className={classes.twoColumnGrid}>
+              <FormTextField {...fields.user_name} label="Username" fullWidth />
+              <ApeTextField
+                label="ETH Address"
+                value={myAddress}
+                disabled
+                fullWidth
+                infoTooltip="Uses the address of your connected wallet"
+              />
+              <FormTextField
+                {...fields.circle_name}
+                label="Circle Name"
+                fullWidth
+              />
+              {myAdminCircles.length ? (
+                <FormAutocomplete
+                  {...fields.protocol_name}
+                  value={fields.protocol_name.value}
+                  onChange={(v: string) => {
+                    const id = protocols.find((p) => p.name === v)?.id;
+                    fields.protocol_id?.onChange(id);
+                    fields.protocol_name.onChange(v);
+                  }}
+                  options={protocols.map((p) => p.name)}
+                  label="Organization Name"
+                  fullWidth
+                  TextFieldProps={{
+                    infoTooltip:
+                      'Add to organizations that you are an admin in or create a new org.',
+                  }}
+                />
+              ) : (
+                <FormTextField
+                  {...fields.protocol_name}
+                  label="Organization Name"
+                  fullWidth
+                  infoTooltip="A circle admin can add to an existing organization."
+                />
+              )}
+              <FormTextField
```

Comment:
gotta hand it to you btw @exrhizo, adding these two fields was stupid-easy. nice work on the form controller!

Reply:
yay!